application/Modules/WebBrowser/WebBrowserModuleConfiguration.cs

```diff
@@ -15,7 +15,7 @@ public class WebBrowserModuleConfiguration : IConfiguration
         ///     Must end in a slash ('/') character.
         ///     Examples for valid values are "60024/" or "60024/johndoe/".
         /// </summary>
-        public string UrlSuffix { get; private set; }
+        public string UrlSuffix { get; protected set; }
```

Comment:
Instead of changing the accessiblity of this, could you please instead create a private subclass in the test and export it with `ComposeExportedValue<WebBrowserModuleConfiguration>` that exposes the setter (or sets a fixed value)?

Reply:
As discussed, there is no decent way around this. Changing it to `protected` is the least invasive option, so we should just stick to this.